src/com/aca/Armine Arzumanyan/HomeWork_1/src/Gadget.java

```diff
@@ -0,0 +1,57 @@
+import java.util.ArrayList;
```

Comment:
redundant _src_ folder in the project and there aren't package names in the files which doesn't allow to compile sources, I guess it has happened because you didn't clone this repo and move your sources from another project directly.